pxr/imaging/hgiVulkan/graphicsPipeline.cpp

```diff
@@ -132,21 +132,19 @@ HgiVulkanGraphicsPipeline::HgiVulkanGraphicsPipeline(
         vertBufs.push_back(std::move(vib));
     }
 
+    VkPipelineVertexInputDivisorStateCreateInfoEXT vertexInputDivisor =
```

Comment:
I'm guessing this work was done on an earlier version of the dev/feature branch, since I believe this change is no longer needed.